tests/unit/nupic/research/spatial_pooler_boost_test.py

```diff
@@ -193,19 +207,22 @@ def verifySDRProperties(self):
     self.assertTrue(_areAllSDRsUnique(self.lastSDR), "All SDR's are not unique")
 
     # Verify that the first two SDR's have some overlap.
-    self.assertGreater(_computeOverlap(self.lastSDR[0], self.lastSDR[1]), 9,
-                       "First two SDR's don't overlap much")
+    self.assertGreater(_computeOverlap(self.lastSDR[0], 
+                       self.lastSDR[1]),
+                       SIMILAR_SDR_OVERLAP,
```

Comment:
The question is: for "default params test", should the inputs (with 50% overlap) still pass the test? I believe they should